patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
+-                    return (String) entry.getKey() + ":" + entry.getValue();
+-                }).sorted().collect(Collectors.joining(",")));
++                List<String> toSort = new ArrayList<>();
++                for (Map.Entry<String, MinecraftKey> entry : multimap.entries()) {
++                    String s = entry.getKey() + ":" + entry.getValue();
++                    toSort.add(s);
++                }
++                toSort.sort(null);
++                StringJoiner joiner = new StringJoiner(",");
++                for (String s : toSort) {
++                    joiner.add(s);
++                }
++                throw new IllegalStateException("Missing required tags: " + (String) joiner.toString());
+             }
+         }, executor1);
+     }
+diff --git a/src/main/java/net/minecraft/server/TickListChunk.java b/src/main/java/net/minecraft/server/TickListChunk.java
+index 8af1229c3da63a838b0bec1cafde1e41cf5cf551..7ee8534bc656e117fa2a21a39693d53ac8a776b8 100644
+--- a/src/main/java/net/minecraft/server/TickListChunk.java
++++ b/src/main/java/net/minecraft/server/TickListChunk.java
+@@ -84,9 +84,9 @@ public class TickListChunk<T> implements TickList<T> {
+     }
+ 
+     public void a(TickList<T> ticklist) {
+-        this.a.forEach((ticklistchunk_a) -> {
++        for (TickListChunk.a<T> ticklistchunk_a : this.a) {
+             ticklist.a(ticklistchunk_a.a, ticklistchunk_a.d, ticklistchunk_a.b, ticklistchunk_a.c);
+-        });
++        }
+     }
+ 
+     static class a<T> {
+diff --git a/src/main/java/net/minecraft/server/VillagePlaceSection.java b/src/main/java/net/minecraft/server/VillagePlaceSection.java
+index 3aaba7fcaf83a491b9da60d156b7f2379f5a6aeb..5151876951d0089fa2ae229c7fee6e418fea3636 100644
+--- a/src/main/java/net/minecraft/server/VillagePlaceSection.java
++++ b/src/main/java/net/minecraft/server/VillagePlaceSection.java
+@@ -52,7 +52,9 @@ public class VillagePlaceSection {
+         this.c = Maps.newHashMap();
+         this.d = runnable;
+         this.e = flag;
+-        list.forEach(this::a);
++        for (VillagePlaceRecord villagePlaceRecord : list) {
++            a(villagePlaceRecord);
++        }
+     }
+ 
+     public Stream<VillagePlaceRecord> a(Predicate<VillagePlaceType> predicate, VillagePlace.Occupancy villageplace_occupancy) {
+diff --git a/src/main/java/net/minecraft/server/VillagePlaceType.java b/src/main/java/net/minecraft/server/VillagePlaceType.java
+index 1fb87bf395ced15bc3ef383dae46a42c5e4d2547..5be0e3485bb7a398f4abbaf1a3b6fc57e827aa8e 100644
+--- a/src/main/java/net/minecraft/server/VillagePlaceType.java
++++ b/src/main/java/net/minecraft/server/VillagePlaceType.java
+@@ -115,13 +115,13 @@ public class VillagePlaceType {
+     }
+ 
+     private static VillagePlaceType a(VillagePlaceType villageplacetype) {
+-        villageplacetype.C.forEach((iblockdata) -> {
```

Comment:
Not a stream.

https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/lang/Iterable.java#l72